mappings/net/minecraft/client/gui/GuiMainMenu.mapping

```diff
@@ -2,11 +2,23 @@ CLASS bgu net/minecraft/client/gui/GuiMainMenu
 	FIELD A SPLASHES_LOC Lkp;
 	FIELD B LOGO_LOC Lkp;
 	FIELD C PANORAMA_LOC [Lkp;
+	FIELD D outdatedGlTextWidth I
```

Comment:
Yes, but can one use it for different kinds of text? Yes. It's not hardcoded /per se/.